server.js

```diff
@@ -66,13 +76,13 @@ var serverLeaveCluster = function (socket, respond) {
   });
 
   respond && respond();
-  console.log(`Server ${socket.instanceId} at address ${socket.instanceIp} on port ${socket.instancePort} left the cluster`);
+  (LOG_LEVEL >= 3) && console.log(`Server ${socket.instanceId} at address ${socket.instanceIp} on port ${socket.instancePort} left the cluster`);
```

Comment:
There are a lot of these, maybe you can create a custom `logInfo` function (or another name) which checks the log level and calls console.log. Then always use `logInfo()` instead of `console.log()` directly. Also the same should be done for console.error and `console.warning()`.

Also maybe also use `if` statement for readability?
Right now the only time this pattern is used in SC is for `callback && callback()`.

Reply:
Instead of inventing a `logInfo()`, with log levels and something else, it'd be better to add a logger dependency, I believe. Even though it'd be an overkill for scc-state, it might be in place for other sc-* and scc-*, or maybe not, who knows :)
Anyway I don't feel like adding a new heavy dependency to all sc-* and scc-* just to reimplement logging. Because logging in sc[c]-*, in general, is good enough already IMO.
So I simply have changed `a && b` to if statements for consistency.